tm/tm.config.js

```diff
@@ -0,0 +1,2 @@
+const key = 'dijuQaQgNxRJLxYmi5ZlROnW7RG0c4kC';
```

Comment:
@tedgreen29 This is a public repo. I don't think you want to check-in your key 😃

Reply:
I could have gitignored the file, but I wanted you guys to be able to use this for now. By the end of the project, I plan to remove this file and provide instructions on how to fill it.

Reply:
Thats ok if you want it to be in there. Just wanted to make sure!